packages/react-native-bpk-component-dialog/src/NativeDialog.ios.js

```diff
@@ -47,6 +47,11 @@ export type State = {
   ...$Exact<CommonProps>,
 };
 
+const isArrayEqual = (a: Array<any>, b: Array<any>) =>
+  _(a)
+    .differenceWith(b, _.isEqual)
```

Comment:
This compares functions with strict equality `===`  which means if you are creating the function in render e.g.

```javascript
<BpkDialog
  {...otherProps}
  actions={[
    {
      text: "Accept",
      type: BUTTON_TYPE.primary,
      callback: () => console.log("Tapped Accept")
    }
  ]}
/>;
```

It will unsub on every render

Reply:
Ok. Then I will have to create a custom method to compare actions 🙂

Reply:
There's not a good way to compare to functions, the best way to ensure they are equals is doing `.toString()` and then compare them.